### PR TITLE
Fuzzy search

### LINKED ISSUE #411: add searcher to bit-dev
https://app.clickup.com/t/2hrjanx

as requested by @ranm8, need to add a searcher to bit-dev.
should reuse code from the `command-bar` aspect in bit-bin (without the aspect)

### PR DESCRIPTION
**Pull request by KutnerUri**
closed #411 
searcher for bit-dev

### CODE REVIEW

File: community/apps/bit-dev/command-bar/command-bar.module.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+.title {
+  margin-right: 4px;
+  max-width: 100%;
+  flex: none;
+}
+
+.description {
+  font-size: 0.85em;
+  color: var(--on-background-low-color);
```
Comment (JoshK2) [inline community/apps/bit-dev/command-bar/command-bar.module.scss:26]: Maybe add default color?

**Review by JoshK2 — APPROVED** [review]
Looks good!
image

File: community/apps/bit-dev/command-bar/command-bar.module.scss (hunk 1 above)
Comment (KutnerUri) [reply community/apps/bit-dev/command-bar/command-bar.module.scss:26]: right

Comment (KutnerUri) [reply community/apps/bit-dev/command-bar/command-bar.module.scss:26]:
```suggestion
  color: var(--on-background-low-color, #9598A1);
```

**Pull request merged**